### PR TITLE
Make room for columns in `linkerd top`

### PR DESCRIPTION
**Pull request by alpeb**
Not only there wasn't enough padding, but also column data were bleeding out into other columns.
I set the column width dynamically upon each table update, taking into account the row with the longest data so everything fits.

### CODE REVIEW

File: cli/cmd/top.go [hunk 1]
```diff
@@ -348,30 +348,44 @@ func renderHeaders(withSource bool) {
...
+		adjustedColumnWidths[1] = max(adjustedColumnWidths[1], runewidth.StringWidth(row.destination))
+		adjustedColumnWidths[3] = max(adjustedColumnWidths[3], runewidth.StringWidth(row.by))
+
+	}
 	for i, row := range *table {
 		x := 0
 		if withSource {
-			tbprint(x, i+headerHeight, row.source)
-			x += columnWidths[0] + 1
+			tbprint(x, i+headerHeight, runewidth.Truncate(row.source, adjustedColumnWidths[0], ""))
```
Comment (adleong) [inline cli/cmd/top.go]: If we are adjusting the column width to the maximum runewidth in that column, is this truncation needed?

**Comment by alpeb** [comment]
Oops you're right. That was a leftover from a previous iteration. I've just removed it.

File: cli/cmd/top.go (hunk 1 above)
Comment (alpeb) [reply cli/cmd/top.go]: Oops you're right. That was a leftover from a previous iteration. I've just removed it.

**Comment by alpeb** [comment]
@adleong Cool. I've forced-pushed that last commit again with the signoff :+1:

**Comment by wmorgan** [comment]
Thanks @alpeb!

**Pull request merged**